fork.yaml

```diff
@@ -327,6 +327,12 @@ def:
     - title: "Geth extras"
       description: Extend the tools available in geth to improve external testing and tooling.
       sub:
+        - title: Logger improvements
+          description: |
+            Extend the logger with context-logging support
+          globs:
+            - "log/logger.go"
+            - "log/root.go"
```

Comment:
Should we also add `internal/testlog/testlog.go` and the test files that were improved as well?

Reply:
Will add `testlog` :+1: